### PR TITLE
Routes panel (Bottom half)

### PR DESCRIPTION
**Pull request by encX**
Resolves: #8 

### Changes in this PR
- Rename `TopHalf` -> `SearchPanel`
- Rename `BottomHalf` -> `RoutesPanel`
- Rename `style.scss` -> `App.scss`
- Display each routes as tab
- Each route contain time, name and canvas shown below
- Add rendering test for App (root component)
- Add shallow snapshot test for new components
- CSS class naming using BEM standard http://getbem.com/introduction/

Screen Shot 2020-08-23 at 5 23 01 PM

### CODE REVIEW

File: client/src/components/RouteDisplay/RouteSummary/RouteSummaryItem.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
+@import "src/css/Common";
+
+.row-summary-item {
```
Comment (nantapong44) [inline client/src/components/RouteDisplay/RouteSummary/RouteSummaryItem.scss:3]: Do you intend to name it like this or just wrong tpying ?

Comment (encX) [reply client/src/components/RouteDisplay/RouteSummary/RouteSummaryItem.scss]: good catch, fixed

Comment (encX) [reply client/src/components/RouteDisplay/RouteSummary/RouteSummaryItem.scss]: I always name the css after the component name. So this one is a mistake.

Comment (nantapong44) [reply client/src/components/RouteDisplay/RouteSummary/RouteSummaryItem.scss]: Okayy

**Review by nantapong44 — APPROVED** [review]

**Pull request merged**